exoctk/tests/test_lightcurve_fitting.py

```diff
@@ -0,0 +1,143 @@
+#! /usr/bin/env python
```

Comment:
@hover2pi You may want to add this module to the `ExoCTK.tests.rst` API docs as well.